pubtools/_pulp/services/pulp.py

```diff
@@ -47,6 +49,17 @@ def add_service_args(self, parser):
             help="Pulp password (or set PULP_PASSWORD environment variable)",
             default=None,
         )
+        group.add_argument(
+            "--pulp-certificate",
+            help="Pulp certificate (or taken from '~/certs/$USER.crt'). "
```

Comment:
The default doesn't seem to be implemented actually, or at least I can't find the code for it (and can't find any code using DEFAULT_CERT_PATH).